lib/livebook/config.ex

```diff
@@ -336,11 +336,11 @@ defmodule Livebook.Config do
   defp parse_connection_config!(config) do
     {node, cookie} = split_at_last_occurrence(config, ":")
 
-    unless node =~ "@" do
-      abort!(~s{expected node to include hostname, got: #{inspect(node)}})
-    end
```

Comment:
I wonder if just removing this check was enough, because the attached mode automatically adds the nodename, or do we really need to `append_hostname` here too?

Reply:
1. I thought that the check must be there for a reason.
2. Also, I don't know how to test `Escripts`, so I couldn't verify if it would work after just removing that check.
3. I couldn't find the `append_hostname` kind of function anywhere, so just replaced the above lines.

Reply:
You might be right. But since I couldn't figure out how to test my changes, I don't think I'm the right person to create the PR for this. 😅

Reply:
Removed the `append_hostname` function.

Reply:
You don't need an escript. The env vars are also used on mix phx.server, so you can do this:

    LIVEBOOK_DEFAULT_RUNTIME=attached:foo:jldkasjdksalk mix phx.server

But you can also call mix escript.build to build it!